fsd_config/form_jsons/cof25/en/community-benefits-cof-25.json

```diff
@@ -78,7 +60,28 @@
                 {
                     "path": "/summary"
                 }
-            ]
+            ],
+            "section": "PTOBPV"
+        },
+        {
+            "path": "/community-benefits",
+            "title": "Community benefits",
+            "components": [
+                {
+                    "type": "Para",
+                    "content": "<label class=\"govuk-!-font-weight-bold\">In this section, we'll ask about:\n</label>\n\n<ul class=\"govuk-list govuk-list--bullet\">\n<li>community benefits project will deliver</li>\n<li>how you\u2019ll measure benefits</li>\n<li>how benefits will be delivered and sustained</li>\n<li>how community ownership will be inclusive</li>\n</ul>",
+                    "options": {},
+                    "schema": {},
+                    "name": "lLCjXj"
+                }
+            ],
+            "next": [
+                {
+                    "path": "/benefits-youll-deliver"
+                }
+            ],
+            "section": "FabDefault",
+            "controller": "start.js"
```

Comment:
I think controller name is `"controller": "./pages/start.js"`

Reply:
It works both ways though

Reply:
![Uploading Screenshot 2024-10-24 at 15.42.47.png…]()